README.md

```diff
@@ -1,33 +1,31 @@
 # RQT2
 
-## Install ROS2
+## Setup
 
-In order to reproduce our setup follow the instructions on the ros2 wiki for [installing ROS2 from source on Ubuntu 18.04](https://index.ros.org/doc/ros2/Linux-Development-Setup/).
+In order to reproduce our setup follow the instructions on the ros2 wiki for [installing ROS2 from source on Ubuntu 18.04](https://index.ros.org/doc/ros2/Linux-Development-Setup/) stopping before the section: *Get ROS 2.0 code*.
 
 TODO: After Crystal release, support building RQT2 on top of debian ROS2 packages
 
 ## Build From Source
 
-### Install Dependencies
-
-This section is only necessary if you haven't install ROS2 from source:
+### Download ROS2 and RQT2 Repositories
 
-    sudo apt-get install python3-vcstool python3-rosdep2
+    mkdir -p ~/ros2_ws/src
+    cd ~/ros2_ws
+    wget https://raw.githubusercontent.com/ros2/ros2/release-latest/ros2.repos
+    wget https://raw.githubusercontent.com/PickNikRobotics/rqt2_setup/master/rqt2.repos
+    vcs import src --force < ros2.repos
+    vcs import src --force < rqt2.repos
 
-If you haven't already, initialize rosdep:
+### Install Dependencies
 
     sudo rosdep init
     rosdep update
+    rosdep install --from-paths src --ignore-src --rosdistro bouncy -y --skip-keys "console_bridge fastcdr fastrtps libopensplice67 rti-connext-dds-5.3.1 urdfdom_headers rqt_gui_cpp"
 
-### Download RQT2 repositories and dependencies
+**Note:** If you see the error message about mulitply packages named *uncrustify_vendor* then run `rm -rf ~/ros2_ws/src` and start this section again.
```

Comment:
*multiple*

>  start this section again.

Which section? Also, is there really no better workaround? If not, explain why you must delete everything e.g. "Because colcon doesn't currently support removing duplicate packages once they have started to be built" or something.